src/sentry/utils/pytest/relay.py

```diff
@@ -0,0 +1,88 @@
+# Fixutres used to interact with a test Relay server
+
+from __future__ import absolute_import
+import pytest
+from os import path, makedirs
+
+from six.moves.urllib.parse import urlparse
+import shutil
+import datetime
+
+from sentry.runner.commands.devservices import get_docker_client
+
+
+def _relay_server_container_name():
+    return "sentry_test_relay_server"
+
+
+def _get_template_dir():
+    return path.abspath(path.join(path.dirname(__file__), "template"))
+
+
+def _remove_container_if_exists(docker_client, container_name):
+    try:
+        container = docker_client.containers.get(container_name)
+    except Exception:
+        pass  # container not found
+    else:
+        container.stop()
+        container.remove()
+
+
+@pytest.fixture(scope="session")
+def relay_server(live_server):
+    now = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S_%f")
+    config_path = path.expanduser("~/.tmp/test_relay_config/{}".format(now))
```

Comment:
Not everyone will like to have new directories in their home root. Why not a temp directory in the project root?

Reply:
we delete the directories at test teardown... 
I don't want to set it relative to the project root because I have no control about where the user is going to clone the project and I want to make sure that the tmp file is in a directory that is (by default) one of the FileSharing directories set up by docker, they are: ( `/Users`, `/Volumes`, `/private` and `/tmp`). We could use `/tmp` if you prefer.

Reply:
I think there's a veeery high chance that users will be cloning the project somewhere inside `/Users`.
`/tmp` might be a bit dangerous because files are regularly deleted from there when not accessed (https://superuser.com/questions/187071/in-macos-how-often-is-tmp-deleted), but this is probably not our case.

Reply:
I concur with Anton on this:
 - either a gitignored directory in the sentry project 
 - or inside the .sentry config directory, which is where other configs live